prime-router/src/test/kotlin/history/azure/SubmissionFunctionIntegrationTests.kt

```diff
@@ -237,6 +240,9 @@ class SubmissionFunctionIntegrationTests {
                     log(ActionLog(InvalidParamMessage("log"), type = ActionLogLevel.warning))
                     reportGraphNode {
                         action(TaskAction.route)
+                        reportGraphNode {
```

Comment:
Should there be another assertion change?

Reply:
I took a look and did not think of any in the context of status/history.

I was thinking maybe an assertion to ensure the lineage is "terminated" so to to speak, but that would just assert against data we inserted.